packages/block/src/block.ts

```diff
@@ -559,48 +559,48 @@ export class Block {
   }
 
   /**
-   * Validates that data gas fee for each transaction is greater than or equal to the
-   * dataGasPrice for the block and that total data gas in block is less than maximum
-   * data gas per block
+   * Validates that blob gas fee for each transaction is greater than or equal to the
+   * blobGasPrice for the block and that total blob gas in block is less than maximum
+   * blob gas per block
    * @param parentHeader header of parent block
    */
   validateBlobTransactions(parentHeader: BlockHeader) {
     if (this.common.isActivatedEIP(4844)) {
-      const dataGasLimit = this.common.param('gasConfig', 'maxDataGasPerBlock')
-      const dataGasPerBlob = this.common.param('gasConfig', 'dataGasPerBlob')
-      let dataGasUsed = BigInt(0)
+      const blobGasLimit = this.common.param('gasConfig', 'maxblobGasPerBlock')
+      const blobGasPerBlob = this.common.param('gasConfig', 'blobGasPerBlob')
+      let blobGasUsed = BigInt(0)
 
       for (const tx of this.transactions) {
         if (tx instanceof BlobEIP4844Transaction) {
-          const dataGasPrice = this.header.getDataGasPrice()
-          if (tx.maxFeePerDataGas < dataGasPrice) {
+          const blobGasPrice = this.header.getblobGasPrice()
+          if (tx.maxFeePerblobGas < blobGasPrice) {
             throw new Error(
-              `blob transaction maxFeePerDataGas ${
-                tx.maxFeePerDataGas
-              } < than block data gas price ${dataGasPrice} - ${this.errorStr()}`
+              `blob transaction maxFeePerblobGas ${
+                tx.maxFeePerblobGas
+              } < than block blob gas price ${blobGasPrice} - ${this.errorStr()}`
             )
           }
 
-          dataGasUsed += BigInt(tx.versionedHashes.length) * dataGasPerBlob
+          blobGasUsed += BigInt(tx.versionedHashes.length) * blobGasPerBlob
 
-          if (dataGasUsed > dataGasLimit) {
+          if (blobGasUsed > blobGasLimit) {
             throw new Error(
-              `tx causes total data gas of ${dataGasUsed} to exceed maximum data gas per block of ${dataGasLimit}`
+              `tx causes total blob gas of ${blobGasUsed} to exceed maximum blob gas per block of ${blobGasLimit}`
             )
           }
         }
       }
 
-      if (this.header.dataGasUsed !== dataGasUsed) {
+      if (this.header.blobGasUsed !== blobGasUsed) {
         throw new Error(
-          `block dataGasUsed mismatch: have ${this.header.dataGasUsed}, want ${dataGasUsed}`
+          `block blobGasUsed mismatch: have ${this.header.blobGasUsed}, want ${blobGasUsed}`
         )
       }
 
-      const expectedExcessDataGas = parentHeader.calcNextExcessDataGas()
-      if (this.header.excessDataGas !== expectedExcessDataGas) {
+      const expectedExcessblobGas = parentHeader.calcNextExcessblobGas()
```

Comment:
```suggestion
      const expectedExcessBlobGas = parentHeader.calcNextExcessBlobGas()
```
just to maintain consistency